Header header
uint8 status
float32 current_speed
float32 current_steering
float32 target_speed
float32 target_steering